source/controlTypes/formatFields.py

```diff
@@ -99,3 +99,83 @@ def translateFromAttribute(fontSize: str) -> str:
 				return FontSize._unitToTranslatableString[measurementUnit] % measurement
 		log.debugWarning(f"Unknown font-size value, can't translate '{fontSize}'")
 		return fontSize
+
+
+class TextAlign(DisplayStringStrEnum):
+	"""Values to use for 'text-align' NVDA format field.
+	These describe the horizontal paragraph position.
+	"""
+	UNDEFINED = 'undefined'
+	LEFT = 'left'
+	CENTER = 'center'
+	RIGHT = 'right'
+	JUSTIFY = 'justify'
+	DISTRIBUTE = 'distribute'
+	CENTER_ACROSS_SELECTION = 'center-across-selection'
+	GENERAL = 'general'
+	FILL = 'fill'
+
+	@property
+	def _displayStringLabels(self):
+		return _textAlignLabels
+
+
+#: Text to use for 'text-align format field values. These describe the horizontal position of a paragraph
+#: or a cell's content.
+_textAlignLabels: Dict[TextAlign, str] = {
+	TextAlign.UNDEFINED: "",  # There is nothing to report if no alignment is defined.
+	# Translators: Reported when text is left-aligned.
+	TextAlign.LEFT: _("align left"),
+	# Translators: Reported when text is centered.
+	TextAlign.CENTER: _("align center"),
+	# Translators: Reported when text is right-aligned.
+	TextAlign.RIGHT: _("align right"),
+	# Translators: Reported when text is justified.
+	# See http://en.wikipedia.org/wiki/Typographic_alignment#Justified
+	TextAlign.JUSTIFY: _("align justify"),
+	# Translators: Reported when text is justified with character spacing (Japanese etc)
+	# See http://kohei.us/2010/01/21/distributed-text-justification/
+	TextAlign.DISTRIBUTE: _("align distributed"),
+	# Translators: Reported when text is centered across multiple cells.
+	TextAlign.CENTER_ACROSS_SELECTION: _("align center across multiple cells"),

```

Comment:
Should this say "centered across selection" instead? it's shorter and handles the generic case better. we may want to use `CENTER_ACROSS_SELECTION` in other contexts to excel